test/string_method.spec.js

```diff
@@ -200,6 +200,56 @@ _ddiast.stringSubstring(__datadog_test_1.call(__datadog_test_0, 2), __datadog_te
     })
   })
 
+  describe('concat (method that allows literals)', () => {
+    it('does not modify String.prototype.concat call if all args are literals', () => {
+      const js = 'String.prototype.concat.call("hello", "world");'
```

Comment:
could we add some test to check that it works also with non string values?
```
String.prototype.concat.call("hello", "world", 2, undefined, null, ...)

Reply:
added